pkg/k8s/watchers/pod.go

```diff
@@ -782,6 +793,27 @@ func (k *K8sWatcher) deleteHostPortMapping(pod *slim_corev1.Pod, podIPs []string
 	return nil
 }
 
+func deepEqualBackends(backends1 []*loadbalancer.Backend, backends2 []*loadbalancer.Backend) bool {
```

Comment:
AFAIU this function doesn't care about backend order in provided slices, can you add a comment about that?

Reply:
Done!